arkiverer/src/test/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/utils/ContainerizedKafka.kt

```diff
@@ -1,48 +1,90 @@
 package no.nav.soknad.arkivering.soknadsarkiverer.utils
 
-import no.nav.soknad.arkivering.soknadsarkiverer.config.*
+import no.nav.security.token.support.client.spring.ClientConfigurationProperties
+import no.nav.soknad.arkivering.soknadsarkiverer.kafka.KafkaConfig
+import org.apache.commons.compress.harmony.pack200.PackingUtils.config
```

Comment:
There are some unused imports that I cant resolve. Could you right-click on the file in the list and select Optimize Imports, please? :)

You could also right-click and do a Reformat Code